assets/src/data/eventespresso/core/actions.js

```diff
@@ -1,12 +1,20 @@
 /**
- * Returns an action object used to update the store with the provided entities.
+ * Action for receiving entity records.
+ * Entity records received will NOT replace any existing entities in the state
```

Comment:
Yes.  That's why `keepExistingEntitiesInObject` (the method you left your other comment on) is _named_ the way it is because it's purpose is to favor existing entities in object.   Thus its the appropriate method to do in this implementation.  Who knows, we may find a case in the future where we want to keep _incoming_ entities in object in which case I might create a new function called (`keepIncomingEntitiesInObject`).